iis/README.md

```diff
@@ -125,18 +86,40 @@ Here's an example of configuration that would check the current machine and a re
     ```
 
     Change the `path` and `service` parameter values and configure them for your environment.
-    See the [sample iis.d/conf.yaml][6] for all available configuration options.
+    See the [sample iis.d/conf.yaml][5] for all available configuration options.
+
+3. [Restart the Agent][6].
+
+#### Containerized
+
+For containerized environments, see the [Autodiscovery Integration Templates][7] for guidance on applying the parameters below.
```

Comment:
Notice that in the Autodiscovery Integration Templates page the <INTEGRATION_NAME> parameter is called out in the first section, but is actually referred to as the <CHECK_NAME> parameter in the configuration section with the examples below. I think we must fix this ASAP because this page is the only source of truth that all the integration pages refer to, and there's no clear link between the two, in which without (this link) users may not be able to use the integrations

which is the only page where we actually explain about the

Reply:
https://github.com/DataDog/documentation/pull/6282